csm_api_client/service/__init__.py

```diff
@@ -25,29 +25,6 @@
 Client for querying the API gateway.
 """
 # Import these names so that they can still be imported from the apiclient package directly
```

Comment:
You're correct, that is not entirely accurate. I actually question whether having things be importable from `csm_api_client.service` directly is all that useful anyway, since that was initially done for compatibility with existing SAT code which imported from `sat.apiclient`. Since all usage of that will be removed, importing the client classes here is not necessary anymore.